content/hardware/08.kits/maker/plant-watering-kit/tutorials/iot-smart-garden-setup-with-nanorp2040/content.md

```diff
@@ -61,7 +61,11 @@ Now on to the circuit assembly! Begin by grabbing the Arduino Nano RP2040 Connec
 
 ![Nano mounted on the adapter](./assets/step1.png)
 
-Then find the relay module, and a grove cable. You will need to cut the grove cable in half to expose the copper cable inside so you can connect it to the Screw terminal adapter. Connect the wires from the grove cable to the Nano Screw Terminal Adapter as follows:
+Then find the relay and a Grove cable. Taking a closer look at the Grove cable you will see that one side is a 3 pin connector (this goes into the moisture sensor), while the other end is a 4 pin connector (this will be cut off in the next step, we won’t be needing that). 
```

Comment:
I think this change is good but it shouldn't go here. This part is talking about the relays Grove cable which is _not_ the cable that has different connectors on each end.
I'd revert this change and move the comment to the section where the moisture sensor is hooked up.